### ISSUE TITLE
Spring semester update 11

### ISSUE THREAD

**Issue by akzysk** [gw-cs-sd/sd-2017-dolphin-image-processor#19]
Working on implementing a way to upload an image and classify it straight from the upload.

Looks like this can be done by saving the samples' segments to the DB with bloodStatus as "notBlood" (arbitrary), evaluating its segments in a test set against many other segments in a pre-determined training set, and then updating the bloodStatus based on the result. This will require some more engineering.

In MySQLCon.java, need to add an SQL query to update the Sample table (in addition to the segment table which is already implemented).
